extensions/cornerstone/src/services/ParametricMapService/index.ts

```diff
@@ -0,0 +1 @@
+export { ParametricMapService } from './ParametricMapService'
```

Comment:
seems like you forgot to commit the service? or maybe you don't need the service anymore?